### PR TITLE
Add Bob and Mode networks

### PR DESCRIPTION
**Pull request by Inkvi**
Had to update optimism and mode chain files to match the latest `distOutput.json`

## Release Notes

- **New Features**
  - Introduced configuration for two new mainnets: "Mode Mainnet" and "Bob Mainnet," enhancing network options.
  
- **Updates**
  - Updated existing configurations for the OP Mainnet and Base network, including changes to dispatcher addresses and client settings.
  - Adjusted explorer entries for better organization and visibility.

These changes improve network accessibility and streamline user interactions with the blockchain.

### CODE REVIEW

File: chains/eip155-10.json [hunk 1]
```diff
@@ -17,21 +17,21 @@
     "decimals": 18
   },
   "infoURL": "https://optimism.io",
-  "shortName": "oeth",
+  "shortName": "optimism",
```
Comment (RnkSngh) [inline chains/eip155-10.json:20]: Are you sure this is the canonical shortname for optimism? e.g. safe url prefix is oeth

**Review by RnkSngh — APPROVED** [review]
lgtm aside from small comment

File: chains/eip155-10.json (hunk 1 above)
Comment (Inkvi) [reply chains/eip155-10.json:20]: I am not sure. To reduce the scope I only made updates to existing chains that were missing. Here is the original [commit](https://github.com/polymerdao/polymer-registry/commit/6b6d75d3217e02049c6b274e129acabf58c70a2b) that changed the short name

**Review by dshiell — APPROVED** [review]
lgtm

**Pull request merged**